src/xml/zcl_abapgit_xml.clas.abap

```diff
@@ -5,12 +5,14 @@ CLASS zcl_abapgit_xml DEFINITION
 
   PUBLIC SECTION.
     METHODS:
-      constructor.
+      constructor
+        IMPORTING iv_filename TYPE string OPTIONAL.
 
   PROTECTED SECTION.
     DATA: mi_ixml     TYPE REF TO if_ixml,
           mi_xml_doc  TYPE REF TO if_ixml_document,
-          ms_metadata TYPE zif_abapgit_definitions=>ty_metadata.
+          ms_metadata TYPE if_abapgit_definitions=>ty_metadata,
```

Comment:
syntax error

Reply:
thx for finding and fixing the syntax error!